spec/lib/site_setting_extension_spec.rb

```diff
@@ -104,6 +109,30 @@
       expect(settings2.hello).to eq(99)
     end
 
+    it "publishes changes across processes" do
```

Comment:
Introducing a test to make sure the actual site settings refresh across process behaviour works as expected